meta.json

```diff
@@ -21,5 +21,10 @@
       "model": "erh:viamrtsp:rtsp-mjpeg"
     }
   ],
-  "entrypoint": "bin/viamrtsp"
```

Comment:
Can you confirm with someone on the registry team that changing the entrypoint between different module versions is safe (I think it is but just want to confirm)?

Reply:
Confirmed it should be fine